src/librustc_hir/hir.rs

```diff
@@ -1898,7 +1898,7 @@ pub enum ImplItemKind<'hir> {
     /// of the expression.
     Const(&'hir Ty<'hir>, BodyId),
     /// A method implementation with the given signature and body.
```

Comment:
The comment here is not changed, even if the one for `TraitItemKind` is (I think this is from the other PR, actually).

Reply:
r=me with the comment changed.